west.yml

```diff
@@ -68,7 +68,7 @@ manifest:
       revision: b4d31f33238713a568e23618845702fadd67386f
       path: modules/hal/nuvoton
     - name: hal_nxp
-      revision: 73445ce7474ffb0eb6fd795eeaac1fbf507e4b7e
+      revision: pull/97/head
```

Comment:
The hal PR has been merged, so you can now update this to the new SHA

Reply:
done